src/platform/macos.rs

```diff
@@ -126,12 +126,18 @@ impl From<ActivationPolicy> for NSApplicationActivationPolicy {
 
 pub trait CustomMenuItemExtMacOS {
   fn set_native_image(&mut self, native_image: NativeImage);
+
+  fn set_icon(&mut self, icon: Icon);
```

Comment:
this should be defined for `CustomMenuItem` on all platforms in `src/menu.rs`

Reply:
Resolved, please refer to the later commits.